app/SIR_forecast.py

```diff
@@ -17,7 +17,11 @@
 import matplotlib.pyplot as plt 
 from get_data import get_canton_data 
 from sqlalchemy import create_engine
-engine = create_engine("postgresql://epigraph:epigraph@localhost:5432/epigraphhub")
+import os
+from dotenv import load_dotenv
+load_dotenv('../.env')
+
+engine = create_engine(f'postgresql://{os.getenv("POSTGRES_USER")}:{os.getenv("POSTGRES_PASSWORD")}@{os.getenv("POSTGRES_HOST")}:{os.getenv("POSTGRES_PORT")}/{os.getenv("POSTGRES_DB")}')
```

Comment:
Perhaps you can use the same approach here...
Creating [config.py](https://github.com/thegraphnetwork/EpiGraphHub/blob/a8439c05d24025d76e376ffc19a01a86ed34faab/Data_Collection/CRON_scripts/config.py#L16), to avoid calling connection variables more than once.